docs/release-notes.txt

```diff
@@ -8,6 +8,35 @@
  Version 0.13
 **************
 
+Version 0.13.6
+==============
+
+**Release Date:** October 14, 2020
+
+**Improvements**
+
+- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional.
```

Comment:
What is the behavior if this field is omitted? @shiyuann

Reply:
The previous commit explained it: "They default to the latest agent images.".  Should we put this wording back in @neilconway @armandmcqueen ?

Reply:
Yes I think being more explicit here is useful.

Reply:
sgtm